dgl/src/Window.cpp

```diff
@@ -318,6 +327,13 @@ struct Window::PrivateData {
 #elif defined(DISTRHO_OS_MAC)
         mView   = nullptr;
         mWindow = nullptr;
+# ifndef DGL_FILE_BROWSER_DISABLED
+        if (NSOpenPanel* panel = fOpenFilePanel)
+        {
+            [panel release];
+            fOpenFilePanel = nullptr;
```

Comment:
the null set should be on the line above, otherwise it is pointless to create a local variable for it.